plopfile.js

```diff
@@ -0,0 +1,67 @@
+module.exports = function (plop) {
+  plop.setGenerator("component", {
+    description: "Create a component",
+    prompts: [
+      {
+        type: "input",
+        name: "name",
+        message: "What is this component's name?",
+      },
+      {
+        type: "input",
+        name: "element",
+        message: "HTML element (div is default)",
+        default: "div",
+      },
+    ],
+    actions: [
+      {
+        type: "add",
+        path: "src/components/{{pascalCase name}}/index.tsx",
+        templateFile: "templates/index.tsx.hbs",
+      },
```

Comment:
@ivanms1  
Thanks for setting this up! I didn't know about `plop` but it seems very useful 🥳

Reply:
I love this thing, creating everything from scratch is a pain